core/insertion_marker_manager.ts

```diff
@@ -56,6 +56,8 @@ const DUPLICATE_BLOCK_ERROR =
  * Class that controls updates to connections during drags.  It is primarily
  * responsible for finding the closest eligible connection and highlighting or
  * unhighlighting it as needed during a drag.
+ *
+ * @deprecated v10 - Use an IConnectionPreviewer instead.
```

Comment:
Are you expecting to merge `connection-previewers` into `develop` and see it released as a future `v10.x` release?

Reply:
yes! should go out mid quarter.